examples/neo-place/client/qr.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
question: Do we still need this file? I assume the `index.html` is enough? The QR code is not really needed (anymore)?

Reply:
Fair enough!
One small thing we could do is adding a comment on the top of this file saying that this file is meant for (live) demonstration purposes or similar. But it also works without.